packages/assistify-help-request/client/views/creationDialog/AssistifyCreateRequest.js

```diff
@@ -109,53 +78,26 @@ Template.AssistifyCreateRequest.events({
 		const position = input.selectionEnd || input.selectionStart;
 		const length = input.value.length;
 		document.activeElement === input && e && /input/i.test(e.type) && (input.selectionEnd = position + input.value.length - length);
-		if (input.value) {
-			t.delaySetExpertise(input.value);
-			t.checkExpertise(input.value);
-		} else {
-			t.delaySetExpertise(false);
-			t.expertise.set('');
-		}
-		if (t.expertise.get() && t.requestTitle.get()) {
-			const requestName = `${ t.expertise.get() }-${ t.requestTitle.get() }`;
-			t.invalidTitle.set(!validateRequestName(requestName));
-			t.requestTitleInUse.set(undefined);
-			t.checkRequestName(requestName);
-		}
+		t.expertise.set(input.value);
+		t.validExpertise.set(false);
+		t.expertiseError.set('');
 	},
 	'input #request_title'(e, t) {
 		const input = e.target;
-		//const length = input.value.length;
-		if (input.value !== t.requestTitle.get()) {
-			t.requestTitle.set(input.value);
+		if (!input.value) {
+			t.titleError.set('');
 		} else {
-			t.requestTitle.set(false);
-		}
-
-		if (t.expertise.get() && input.value) {
-			const requestName = `${ input.value }`;
-			t.invalidTitle.set(!validateRequestName(requestName));
-			if (RocketChat.settings.get('UI_Allow_room_names_with_special_chars')) {
-				t.checkRequestDisplayName(requestName);
-			} else {
-				t.checkRequestName(requestName);
-			}
-			// t.invalidTitle.set(!validateRequestName(requestName));
-			// t.checkRequestName(requestName);
-			t.requestTitleInUse.set(undefined);
-		} else if (!input.value) {
-			t.invalidTitle.set(false);
-			t.requestTitleInUse.set(undefined);
+			t.requestTitle.set(input.value);
+			t.debounceValidateRequestName(input.value);
```

Comment:
All the validation should happen inside the validation method - even if it's just about allowing initial values